### DISCUSSION TITLE
What is the plugin for the cursor scope highlighting?

### DISCUSSION THREAD

**Discussion by vabatta** [crivotz/nv-ide#47] [Q&A] [answered]
Specifically in this screenshot from the homepage, I cannot figure out which plugin creates that highlight colouring on the left hand between `def` and `end`.

![](https://raw.githubusercontent.com/crivotz/nv-ide/master/screenshots/nv-ide_screenshot_3.png)

**Comment by crivotz** [comment, marked as answer]
Hi, I removed it and went back to [indent-blankline](https://github.com/lukas-reineke/indent-blankline.nvim) (while still considering hlchunk better as an aesthetic effect) anyway the plugin is as follows:

* [shellRaining/hlchunk.nvim](https://github.com/shellRaining/hlchunk.nvim): This is the lua implementation of nvim-hlchunk, you can use this neovim plugin to highlight your indent line and the current chunk (context) your cursor stayed

and the configuration code used was this:

```lua
{
  "shellRaining/hlchunk.nvim",
  event = { "UIEnter" },
  config = function()
    require('hlchunk').setup({
      chunk = {
        chars = {
          horizontal_line = "━",
          vertical_line = "┃",
          left_top = "┏",
          left_bottom = "┗",
          right_arrow = "━",
        },
      },
      blank = {
        enable = false,
      },
    })
  end
}
```
